assets/panel/panel.js

```diff
@@ -50,6 +50,11 @@ DebuggerPanel.prototype = {
             .setNodeFront(front, { reason: "debugger" });
 
           return Promise.all([onNodeFrontSet, onInspectorUpdated]);
+        }.bind(this),
+        openConsoleAndEvaluate: async function(input) {
+          await this.toolbox.selectTool("webconsole");
```

Comment:
If I read https://searchfox.org/mozilla-central/rev/78e8de968efd77ef3b30ea081bce07614cd6776b/devtools/client/framework/toolbox.js#1952,1956 correctly, you will receive the panel here, so you don't have to do `getPanel` on the next line.